### PR TITLE
Add support for StarCraft

### PR DESCRIPTION
**Pull request by ttuygun**

### CODE REVIEW

**Comment by lra** [comment]
Aren't the save files stored in the cloud?

**Comment by ttuygun** [comment]
> Aren't the save files stored in the cloud?

Some of the settings and achievements save in the cloud but **campaign progress** do now save in the cloud.

File: mackup/applications/starcraft.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = StarCraft
+
+[configuration_files]
+Library/Application Support/Blizzard/StarCraft
+
```
Comment (lra) [inline mackup/applications/starcraft.cfg]: you can remove this extra line.

**Comment by lra** [comment]
> Some of the settings and achievements saved in the cloud but campaign progress do now saved in the cloud.

Sorry, is it saved in the cloud? What's in this folder?

**Comment by ttuygun** [comment]
> > Some of the settings and achievements saved in the cloud but campaign progress do now saved in the cloud.
> 
> Sorry, is it saved in the cloud? What's in this folder?

Yes it saved in the cloud but I don't sure the exact location.

File: mackup/applications/starcraft.cfg (hunk 1 above)
Comment (ttuygun) [reply mackup/applications/starcraft.cfg]: removed

**Comment by lra** [comment]
So why this is needed if settings, achievements and campaigns are saved in the cloud?

**Comment by lra** [comment]
Not relevant, everything seems to be cloud saved.

**Pull request closed without merging**